parachain/src/wasm_executor/mod.rs

```diff
@@ -39,7 +39,7 @@ const MAX_VALIDATION_RESULT_HEADER_MEM: usize = MAX_CODE_MEM + 1024; // 16.001 M
 
 /// The execution mode for the `ValidationPool`.
 #[derive(Clone, Debug)]
-pub enum ExecutionMode {
+pub enum IsolationStrategy {
```

Comment:
Maybe `ValidationIsolationStrategy`? Because even now just reading `IsolationStrategy` will tell you nothing without reading the docs.

Reply:
Sure! But we have to stop somewhere. Initial candidate I started with something like that `Wasm Validation Worker Isolation Strategy`. But I figured that `IsolationStrategy` should be unique enough because it's used in the context `wasm_executor`. Also, I got a bit of PTSD because it reminded me of Java naming culture : D